pkg/syncer/metadatasyncer.go

```diff
@@ -1043,26 +1042,16 @@ func csiPVDeleted(ctx context.Context, pv *v1.PersistentVolume, metadataSyncer *
 		} else {
 			volumeHandle = pv.Spec.CSI.VolumeHandle
 		}
-		if pv.Spec.ClaimRef == nil || pv.Spec.PersistentVolumeReclaimPolicy != v1.PersistentVolumeReclaimDelete {
-			log.Debugf("PVDeleted: Setting DeleteDisk to false")
-			deleteDisk = false
-		} else {
-			// We set delete disk=true for the case where PV status is failed after deletion of pvc
-			// In this case, metadatasyncer will remove the volume
-			log.Debugf("PVDeleted: Setting DeleteDisk to true")
-			deleteDisk = true
-		}
-		log.Debugf("PVDeleted: vSphere CSI Driver is deleting volume %v with delete disk %v", pv, deleteDisk)
 
-		if err := metadataSyncer.volumeManager.DeleteVolume(ctx, volumeHandle, deleteDisk); err != nil {
+		log.Debugf("PVDeleted: vSphere CSI Driver is deleting volume %v with delete disk %v", pv, false)
```

Comment:
No need to use %v and false now. You can just add a log that says that it is removing CNS tag.

Reply:
Makes sense